st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -181,6 +181,12 @@
                                 'doesn\'t finish in timeout seconds.'),
                 'type': 'integer',
                 'default': FABRIC_RUNNER_DEFAULT_ACTION_TIMEOUT
+            },
+            'port': {
```

Comment:
Does this means we don't support `<host>:<port>` notation for the `hosts` argument anymore? Or how is this supposed to work?